packages/boxed-expression-component/src/expressions/ContextExpression/ContextExpression.tsx

```diff
@@ -118,7 +118,7 @@ export function ContextExpression(contextExpression: ContextExpressionDefinition
   const beeTableColumns = useMemo<ReactTable.Column<ROWTYPE>[]>(() => {
     return [
       {
-        accessor: "context-expression" as any, // FIXME: Tiago -> ?
+        accessor: contextExpression.id as any, // FIXME: Tiago -> ?
```

Comment:
People from the future will fix the issue about "accessor" not being correctly typed as string.

Reply:
Agreed :)